hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -226,6 +226,24 @@
         - 'kubernetes-e2e-{suffix}'
 
 - project:
+    name: kubernetes-e2e-gce-1-2
```

Comment:
Not really. I put this in between the master and 1.1 jobs, but there are master branch feature tests elsewhere and it's a bit of a mess.